mantidproject/mantid
# Created workbench tutorial in mantid basics course.

I have written a section for the mantid basic course for using the workbench covering: how to open it, what tools it provides such as the plotting toolbox and the scrippting window, what parts are missing (WIP), and two exersizes on using the script window to create plots that the user had previously made using the GUI in other parts of the tutorial.

**To test:**
1. Build the documentation.
1. Look in tutorials/mantid_basic_course/using_workbench.
1. Check that everything builds correctly and is legible. 

Fixes #25707

*This does not require release notes* because it is a simple change to documentation.


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
